### PR TITLE
backend code

### PR DESCRIPTION
**Pull request by DarkShadow20**

### CODE REVIEW

File: router/cart-router.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+router.route("/")
+.get( async (req,res)=>{
+    try {
+      const cartItems = await CartItem.find().populate("_id");
+      const normalizedCartItems = cartItems.map((item) => {
+         const { _id, ...doc } = item._id._doc;
+        return { id: _id, ...doc, quantity: item.quantity };
+      });
+      res.json(normalizedCartItems);
```
Comment (DanishOnGit) [inline router/cart-router.js]: minor refactor ,good practice: `res.status(200).json({success:true,noramalizedCartItems})`

File: models/products-model.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  discount: Number,
+  category: String,
+  rating:Number,
+  quantity:Number
+});
+
+const Product = mongoose.model("Product", productSchema);
+
+// one time add in db
+async function fillProductsCollection() {
```
Comment (Rupamdas832) [inline models/products-model.js]: Its better you use ES6 syntax everywhere. Like 

const fillProductsCollection = () => { 
}

File: index.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+const cartRouter = require("./router/cart-router");
+const wishlistRouter = require("./router/wishlist-router");
+
+const app = express();
+const port = 4000;
+app.use(cors());
+app.use(bodyParser.json());
+
+connectDB();
+// run once
```
Comment (Rupamdas832) [inline index.js]: You can remove the comments once done.

**Review by Rupamdas832 — CHANGES_REQUESTED** [review]
Nicely written code. All the best practices. No hard changes according to me required.

**Pull request merged**